src/frontend/Komponenter/Behandling/Totrinnskontroll/SendTilBeslutterFooter.tsx

```diff
@@ -66,6 +106,24 @@ const SendTilBeslutterFooter: React.FC<{
             .finally(() => settLaster(false));
     };
 
+    useEffect(() => {
+        sjekkOmFremleggKanOpprettes();
+    }, [sjekkOmFremleggKanOpprettes]);
+
+    const håndterCheckboxEndring = (oppgavetype: FremleggsoppgaveType) => {
```

Comment:
Når man kan benytte forrige state til å bygge den nye staten så skal man benytte `prevState` fra en callBack-funksjon som man sender inn i `setState()`. Så denne funksjonen burde se slik ut:

```suggestion
const håndterCheckboxEndring = (oppgavetype: FremleggsoppgaveType) => {
        if (skalOppretteFremleggsoppgave.includes(oppgavetype)) {
            settSkalOppretteFremleggsoppgave((prevState) =>
                prevState.filter((prevOppgavetype) => prevOppgavetype !== oppgavetype)
            );
        } else {
            settSkalOppretteFremleggsoppgave((prevState) => [...prevState, oppgavetype]);
        }
    };
```